### PR TITLE
[HWORKS-646] Move airflow dags to hopsfs

### PR DESCRIPTION
**Pull request by gibchikafa**

### CODE REVIEW

File: Karamelfile [hunk 1]
```diff
@@ -3,6 +3,12 @@ dependencies:
     global:  
       - ndb::mysqld
       - hops::docker_registry
+      - hops::nm
```
Comment (SirOibaf) [inline Karamelfile:6]: You don't need the dependency to the node manager.

File: Karamelfile [hunk 2]
```diff
@@ -3,6 +3,12 @@ dependencies:
...
+      - hops::dn
+  - recipe: hops_airflow::install
+    global:
+      - hops::nm
+      - hops::dn
```
Comment (SirOibaf) [inline Karamelfile:11]: Karamel doesn't support dependencies for the install recipes so this can be removed.

File: attributes/default.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@
 
 # User configuration
 default['airflow']["version"]         = "1.10.10"
-default['airflow']['url']             = "#{node['download_url']}/airflow/#{node['airflow']['version']}/airflow.tgz"
+default['airflow']['url']             = "https://repo.hops.works/dev/gibson/airflow.tgz"
```
Comment (SirOibaf) [inline attributes/default.rb:22]: Remember to put it back.

File: templates/default/airflow-webserver.service.erb [hunk 4]
```diff
@@ -17,14 +17,18 @@ Description=Airflow webserver daemon
...
+ExecStart = /usr/bin/docker run --rm --name airfow-webserver \
+           -e "IS_TLS=<%= node['hops']['tls']['enabled'] %>" \
+           -e "NAMENODE_IP=<%= @namenode_fdqn %>" \
+           -e "NAMENODE_PORT=<%= node['hops']['nn']['port'] %>" \
            -v <%= node['airflow']['base_dir'] %>/airflow.cfg:/airflow/airflow.cfg \
-           -v <%= node['airflow']['data_volume']['dags_dir'] %>:/airflow/dags \
            -v <%= node['airflow']['data_volume']['log_dir'] %>:/airflow/logs \
            -v <%= node['airflow']['data_volume']['secrets_dir'] %>:/airflow/secrets \
-           --network=host \
+           -v /srv/hops/super_crypto/airflow:/srv/fuse/certs \
```
Comment (SirOibaf) [inline templates/default/airflow-webserver.service.erb:27]: The `airflow` user here should be parameterized and not hardcoded.

File: templates/default/airflow-scheduler.service.erb [hunk 5]
```diff
@@ -18,13 +18,17 @@ After=network.target <%= @deps %>
...
 ExecStart = /usr/bin/docker run --rm --name airfow-scheduler \
+           -e "IS_TLS=<%= node['hops']['tls']['enabled'] %>" \
+           -e "NAMENODE_IP=<%= @namenode_fdqn %>" \
+           -e "NAMENODE_PORT=<%= node['hops']['nn']['port'] %>" \
            -v <%= node['airflow']['base_dir'] %>/airflow.cfg:/airflow/airflow.cfg \
-           -v <%= node['airflow']['data_volume']['dags_dir'] %>:/airflow/dags \
            -v <%= node['airflow']['data_volume']['log_dir'] %>:/airflow/logs \
            -v <%= node['airflow']['data_volume']['secrets_dir'] %>:/airflow/secrets \
-           --network=host \
+           -v /srv/hops/super_crypto/airflow:/srv/fuse/certs \
```
Comment (SirOibaf) [inline templates/default/airflow-scheduler.service.erb:27]: You should parametrize the `airflow` user here.

File: Karamelfile (hunk 2 above)
Comment (gibchikafa) [reply Karamelfile:11]: But when I remove the dependancy, the install recipe was being run before the namenode and was failing. I will retest.

File: Karamelfile (hunk 1 above)
Comment (gibchikafa) [reply Karamelfile:6]: My bad I meant namenode (nn).

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**